Test/CoreSDK.Test/Shared/Extensibility/TelemetryConfigurationFactoryTest.cs

```diff
@@ -295,6 +295,32 @@ public void LoadInstanceInitializesGivenInstanceAndDoesNotRequireSpecifyingTypeA
         }
 
         [TestMethod]
+        public void LoadInstanceHandlesEnumPropertiesWithNumericValue()
+        {
+            var definition = new XElement(
+                "Definition",
+                new XElement("EnumProperty", "3"));
+
+            var original = new StubClassWithProperties();
+            object instance = TestableTelemetryConfigurationFactory.LoadInstance(definition, typeof(StubClassWithProperties), original, null);
```

Comment:
I personally dislike using var for anything except anonymous types and in this case with object I still don't know what the type or interface is that is returned from this factory method.